test/ShapeCrawler.Tests.Unit/ShapeTests.cs

```diff
@@ -582,4 +582,96 @@ public void Text_Setter_updates_shape_text()
         // Assert
         shape.Text.Should().Be("Test");
     }
+
+    [Test]
+    [SlideShape("057_corner-radius.pptx", 1, "Size 1 Round 0.25", "0.20834")]
+    [SlideShape("057_corner-radius.pptx", 1, "Size 2 Round 0.25", "0.20834")]
+    [SlideShape("057_corner-radius.pptx", 1, "Size 3 Round 0.25", "0.20834")]
+    [SlideShape("057_corner-radius.pptx", 1, "Size 1 Round 0", "0.0")]
+    [SlideShape("057_corner-radius.pptx", 1, "Size 1 Round X", "0.35")]
+    [SlideShape("057_corner-radius.pptx", 1, "Size 1 Round 1", "1.0")]
+    [SlideShape("057_corner-radius.pptx", 1, "Size 1 Round 0.75", "0.61112")]
+    public void CornerRadius_getter_returns_values(IShape shape, string expectedRoundednessStr)
+    {
+        // Arrange
+        decimal? expectedRoundedness = string.IsNullOrEmpty(expectedRoundednessStr) ? null : decimal.Parse(expectedRoundednessStr);
+
+        // Act
+        var actualRoundedness = shape.CornerRoundedness;
+
+        // Assert
+        actualRoundedness.Should().Be(expectedRoundedness);
+    }
+
+    [Test]
+    public void CornerRadius_getter_returns_default_values()
+    {
+        // Arrange
+        var pres = new Presentation();
+        var shapes = pres.Slides[0].Shapes;
+        shapes.AddRoundedRectangle(10, 20, 100, 200);
+        var shape = shapes[0];
+
+        // Act
+        var actualRoundedness = shape.CornerRoundedness;
+
+        // Assert
+        // Rounded rectangles with no specified corner roundedness behave as if
+        // the value was set to 0.35.
```

Comment:
I think this explanation is a good candidate for the parameter `because` of the assertion method `.Be()`:

`actualRoundedness.Should().Be(0.35m, "because the Rounded Rectangle with no specified corner roundness behaves as if the value was set to 0.35.");`

Reply:
89e8854d5cdc08c6327552b87711d973edae2df9